### DISCUSSION TITLE
[Question] What do I get at the end of the operations? A screenshot or original file edited?

### DISCUSSION THREAD

**Discussion by mbenci** [hm21/pro_image_editor#91] [Q&A] [answered]
### Platforms

Android, iOS, Web

### Your question

At the end of the various operations on the images what I get is a screenshot of the page on the screen? 
Is it possible to preserve the original size and quality of the image by applying various graphic effects on top?
Thank you

**Comment by hm21** [comment, marked as answer]
Yes, we can compare it a bit like a screenshot, but the “screenshot” respects the pixel ratio of the image and the screen. 

For example, you have a device with a pixel ratio of 1 and a width of 100px. The original width of the image is 2000 pixels. Normally, when you create a screenshot from the device internal system, you will get an image with a width of 100px because the device doesn't read the rendered image pixel ratio. The difference in my editor is that the editor calculates the pixel ratio from your image. So, at last, the editor will create a “screenshot” from an image with the original width of 2000 pixels including all layers.
